src/api/metadataApi.js

```diff
@@ -0,0 +1,22 @@
+/**
+ * Copyright (c) Hathor Labs and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import { requestExplorerServiceV1 } from './axiosInstance';
+
+const metadataApi = {
+  getDag(id) {
+    return requestExplorerServiceV1().get(`metadata/dag`, {params: {id}}).then((res) => {
+      if (id in res.data) {
+        return res.data[id]
+      }
+    }).catch((error) => {
+      // something wrong with request
```

Comment:
We should discuss that. We added in the wallet with the user permission. I'm not sure if we should just add this in the explorer without user approval. It would be good but it should be discussed first